### PR TITLE
Update intro to quick start guide

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

File: vipps-login-api-quick-start.md [hunk 1]
```diff
@@ -7,62 +7,59 @@ END_METADATA -->
...
 1. Click the "eye" icon and, in the dropdown window, click `Edit` in the top-right corner.
 1. Fill in the `Current Value` for the following fields to get started.
-   - `well-known_uri`
-   - `client_id`
-   - `client_secret`
-   - `redirect_uri`
+   * `client_id` - Merchant key is required for getting the access token.
+   * `client_secret` - Merchant key is required for getting the access token.
+   * `well-known_uri` - URL to well-known endpoint for used environment. In the test environment, you can use `https://apitest.vipps.no/access-management-1.0/access/.well-known/openid-configuration`. See [API Guide: well known](vipps-login-api#openid-connect-discovery-endpoint) for more details.
+   * `redirect_uri` - Redirect URL which the user agent is redirected to after finishing a login. The URL must be exactly the same as the one specified on portal.vipps.no. This URI needs to be pre-registered with Vipps and supplied as a query parameter on calls to the OAuth2 authorize endpoint.
```
Comment (rebekaburnett) [inline vipps-login-api-quick-start.md:53]: Is there an example we can give? I also don't know what I can use here, to make it work.

File: vipps-login-api-quick-start.md [hunk 2]
```diff
@@ -7,62 +7,59 @@ END_METADATA -->
...
 ## Make API calls
 
 1. Send request `Get OIDC well-known`.
-1. In Postman environment "vipps-login-api" copy value of key `start_login_uri`, and use this url in any browser.
-1. Finish Vipps login.
-1. Copy query param `code` from url the browser was redirected to after finishing Vipps login. Paste this code into key `code` in Postman environment `vipps-login-api`.
-1. Send request `Get token` to get access token and id token.
-1. Send request `Get user info` to get user info of logged in user.
+1. In your active Postman environment, "vipps-login-api", copy the value of key `start_login_uri`, and use this URL in any browser.
+1. Finish Vipps login. This request includes a scope parameter that requests access to user information, so you may see an extra screen during login.
```
Comment (alfmags) [inline vipps-login-api-quick-start.md:59]: so you may see an extra screen during login.
 
 Rephrase to
 
 This means that if you have not yet consented to sharing your user information a distinct screen asking for your consent will appear the first time.

Comment (rebekaburnett) [reply vipps-login-api-quick-start.md]: That's great! Thanks!

**Review by alfmags — APPROVED** [review]
Looks great to me!

**Pull request merged**